pkg/otel/templates/base.tmpl

```diff
@@ -0,0 +1,67 @@
+{{- define "aggregator-config" -}}
+receivers:
+  otlp:
+    protocols:
+      grpc: {}
+      http: {}
+  {{ template "metrics-self-receiver" . }}
+  {{ template "metrics-prometheus-receiver" . }}
+processors:
+  batch:
+    send_batch_size: 1000
+    timeout: 15s
+  memory_limiter:
+    limit_mib: 1000
+    spike_limit_mib: 250
+    check_interval: 1s
+  transform:
+    log_statements:
+    - context: log
+      statements:
+      - set(attributes["log_type"], "event") where attributes["k8s.event.uid"] != nil
+exporters:
+  otlphttp:
+    endpoint: "{{ .AgentEndpoint }}"
+    tls:
+      insecure: true
+    sending_queue:
+      num_consumers: 4
+      queue_size: 100
+    retry_on_failure:
+      enabled: true
+  {{ template "metrics-remotewrite-exporter" .}}
+service:
+  {{template "metrics-self-telemetry" . }}
+  pipelines:
+  {{template "metrics-remotewrite-pipeline" . }}
+{{- end -}}
+
+{{- define "node-receivers" -}}
+{{- template "metrics-node-receivers" . -}}
+{{- end -}}
+
+{{define "node-config" }}
```

Comment:
Likewise are we actually using this template anywhere?

Reply:
also config validation tests for now

Reply:
These feel a little clunky to me because they're completely disconnected from the actual code.  E.g it's possible to modify the actual templates the code is deploying without changing these and be in a situation where the validation is passing, but the templates themselves won't (unless I'm missing something).  The instinct is right, but I think this is not the right mechanism.

Reply:
yeah, I couldn't come up with anything better atm, open to any suggestions

Reply:
we could move all the templates from pkg/resources/collector into the fs.Embed and have the controller tests build the config maps and drive the config validation there?

Reply:
To be honest I would just remove them until we come up with something better.

Reply:
Started https://github.com/rancher/opni/discussions/1245 to form ideas over a long term solution